### PR TITLE
fixed nginx deployment

### PR DESCRIPTION
**Pull request by jbyers19**
Signed-off-by: Jared Byers <j.byers@f5.com>

**Description**
Updates the NGINX Service Mesh adapter so that it will deploy the mesh successfully.

**Notes for Reviewers**
The root cause of the mesh deployment failures was fixed by https://github.com/meshery/meshkit/pull/174. This PR pulls in the latest version of `meshkit` with the fix.

Changes include:
- `Dockerfile`:
    - removed `GOPROXY` and `GO111MODULE` from build command (no longer necessary in current go version)
    - pull latest version of `kubeopenapi-jsonschema`
- `Makefile`: set correct port for NGINX adapter
- dependency updates
- added some logic to set the default namespace for deployment to `nginx-system`. This is for two reasons: 1. NGINX needs to be in its own namespace, and 2. it is a workaround for https://github.com/meshery/meshery/issues/5395.
- added some logic to disable sidecar auto-injection in the `meshery` namespace

**Edit:** Also updated the workflows so they would pass.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [X] Yes, I signed my commits.

### CODE REVIEW

**Comment by jbyers19** [comment]
I think some of the workflows need to be updated. It is passing `golangci-lint` locally. Will investigate the check failures tomorrow.

File: Makefile [hunk 1]
```diff
@@ -11,7 +11,7 @@ docker:
 docker-run:
 	(docker rm -f meshery-nginx-sm) || true
 	docker run --name meshery-nginx-sm -d \
-	-p 10007:10007 \
+	-p 10010:10010 \
```
Comment (leecalcote) [inline Makefile:14]: Wowser

File: nginx/install.go [hunk 2]
```diff
@@ -76,7 +96,12 @@ func (nginx *Nginx) applyHelmChart(del bool, version, namespace string) error {
 		Action:          act,
 		CreateNamespace: true,
 		ReleaseName:     chart,
-	})
+		OverrideValues:  overrideVal,
+	}
+
+	// Install Helm chart.
+	nginx.Log.Info(fmt.Sprintf("Installing NGINX Service Mesh %s using Helm chart: %+v\n", version, c))
```
Comment (leecalcote) [inline nginx/install.go:103]: Lovely

**Comment by jbyers19** [comment]
Updated the workflows so they would pass. Just had to bump the golang version for the static checks to fix those. Updated the namespace in the e2e tests to `nginx-system` so it would pass. Future versions of NGINX Service Mesh will no longer include grafana, jaeger, and prometheus deployments, so I preemptively removed those checks from the test.

**Review by Revolyssup — APPROVED** [review]
LGTM

**Review by leecalcote — APPROVED** [review]
Looking good.

**Comment by leecalcote** [comment]
🎈 🎈

**Pull request merged**